e2e/davinci-app/main.ts

```diff
@@ -55,7 +54,9 @@ const urlParams = new URLSearchParams(window.location.search);
   if (continueToken) {
     resumed = await davinciClient.resume({ continueToken });
   } else {
-    await Config.setAsync(config);
+    // the current davinci-config has a slightly
+    // different middleware type than the old legacy config
+    await Config.setAsync(config as any);
```

Comment:
Because the new middleware type does slightly differ from the original middleware type, I casted this temporarily.

Thats the only issue with this as it stands